patcher/MainForm.cs

```diff
@@ -54,26 +55,14 @@ private void ToggleTheme(bool darkModeEnabled)
         public MainForm()
         {
             InitializeComponent();
-            
+
             // Try to get ServerList
-            try
-            {
-                var fileContent = File.ReadAllText("Servers.json", Encoding.UTF8);
-                gameServers = JsonConvert.DeserializeObject<GameServer[]>(fileContent);
-                if (gameServers == null || gameServers.Length == 0)
-                {
-                    gameServers = CreateDefautServerList();
-                }
+            string appData = Environment.GetFolderPath(Environment
+                .SpecialFolder
+                .ApplicationData);
 
-            }
-            catch ( FileNotFoundException ex)
-            {
-                gameServers = CreateDefautServerList();
-            }
-            catch (JsonSerializationException ex) 
-            {
-                gameServers = CreateDefautServerList();
-            }
+            configLocation = $@"{appData}\PeacockProject\";
```

Comment:
Please always use `Path.Combine`.

Reply:
I had stolen this line from `Settings.cs` :P